packages/story-utils/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as getStories } from './getStories';
 export { default as prepareStory } from './prepareStory';
+export { default as generateSnapshot } from './generateSnapshot';
```

Comment:
Can you add an entry to the story-utils/README.md, plz 😸

Reply:
thanks for the reminder